R/use_extendr.R

```diff
@@ -125,7 +128,13 @@ use_extendr <- function(path = ".",
 
   edition <- match.arg(edition, several.ok = FALSE)
   cargo_toml_content <- to_toml(
-    package = list(name = crate_name, publish = FALSE, version = "0.1.0", edition = edition),
+    package = list(
+      name = crate_name,
+      publish = FALSE,
+      version = "0.1.0",
+      edition = edition,
+      `rust-version` = "1.65"
```

Comment:
This explicitly specifies the msrv for extendr to ensure that a version 4 cargo.lock file isn't generated.